packages/tools/guiEditor/src/components/commandDropdownComponent.tsx

```diff
@@ -121,6 +121,15 @@ export class CommandDropdownComponent extends React.Component<ICommandDropdownCo
                                             )}
                                         </div>
                                     );
+                                } else if (m.fileButton && m.label === 'Load contorl') {

```

Comment:
nit: There' no need to test for `m.fileButton` as we are in the `else` of a `if (!m.fileButton)`.

Also, I feel a bit uneasy with testing a label in some logic code... Is there no way around it?

Reply:
Hi @Popov72, I think adding another property in "items" can also be use to check for button type for "Load control" button. New property in "items" added in commit to check button
commit: https://github.com/BabylonJS/Babylon.js/commit/ef8410cf2835f23dd355f8baf525218c9a4bca67